Update smtp-email-setup.rst (#3305)
mattermost/docs: source/install/smtp-email-setup.rst

> -  You must run the following commands from the same machine or virtual

Reviewer: It looks like the "following" commands have been removed from this section. Are there still commands that can/must be run? If not, perhaps something like this could work: 

- Ensure you are running commands from the same machine or virtual instance where mattermost/bin/mattermost is located. 
- Attempt to telnet to the email service to make sure the server is reachable.
- Ping the smtp server to see if a connection can be established.

Author: So the problem here, and why I made the change, is that we don't install telnet inside the docker containers so we can't advise using that to verify the connection. 

I think referring to ping is probably the best thing we can do at this point apart from telling people to install telnet themselves and test it that way.

       250-STARTTLS
       250-PIPELINING
       250 8BITMIME
       

.. note:: 

>   We're not installing telnet by default on the official docker images, so you either need to use ``ping`` on those or install telnet yourself either directly in the machine or by modifying the dockerfile.

Reviewer: ```suggestion
  As we're not installing telnet by default on the official docker images you either need to use ``ping`` on those or install telnet yourself either directly modifying the dockerfile.
```

Not sure what machine we're referring to so I removed it - happy to put it back if we can clarify.
